docs/pull_request_template.md

```diff
@@ -0,0 +1,32 @@
+## Type of change
+- [ ] New feature
+- [ ] Bug Fix
+- [ ] Test
+- [ ] Refactor/Style
+
+## Description
+Relevant User Story(s): 
+
+*Brief summary of the change(s) included in this pull request*
+
+## Implements/Fixes:
+
+description closes #
+
+## Check the correct boxes
+- [ ] This broke nothing
+- [ ] This broke some stuff
+- [ ] This broke everything
+
+## Testing Changes
+- [ ] No Tests have been changed
+- [ ] Some Tests have been changed
+- [ ] All of the Tests have been changed
+
+*Please describe what in the world happened if tests are broken*
+
+## Checklist:
+- [ ] My code has no unused/commented out code
+- [ ] I have reviewed my code
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] I have fully tested my code
```

Comment:
I love this template!  t will help us stay consistent in our PR comments and more organized overall. Thank you!